src/DatabaseInfo.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+from ast import literal_eval
+from urllib.request import urlopen
+import json
+from io import StringIO
+
+
+
+# BiGG
+
+# load BiGG as dataframe
+BiGG = pd.read_csv("http://bigg.ucsd.edu/static/namespace/bigg_models_metabolites.txt", sep="\t").fillna("")
+
+# load the file with InChIKey, formulae and charges for metabolites from BiGG API as a dataframe
+data_df = pd.read_csv("BiGG_API.csv").fillna("")
+
+# convert dataframe entries to lists
+data_df["InChIKey"] = data_df["InChIKey"].apply(lambda x: literal_eval(x) if x != "" else x)
+data_df["formulae"] = data_df["formulae"].apply(literal_eval)
+data_df["charges"] = data_df["charges"].apply(literal_eval)
+
+# skip names that don't exist as metabolites in BiGG API
+BiGG = BiGG[~BiGG["universal_bigg_id"].isin(["Recon3D", "iYS854"])].reset_index()

```

Comment:
Why do you exclude Recon3D and iYS854 from the list here?

Reply:
3 metabolites had their info columns shuffled and I was excluding them, but in the new version I fixed their column order and they are included.